<launch>
    <arg name="visualization" default="true"/>

    <node pkg="terrain_navigation_ros" type="visualize_path_node" name="test_planner" output="screen">
        <param name="file_path" value="/home/jaeyoung/dev/mesh/groundtruth_roi_meshlab.obj"/>
        <param name="output_file_path" value="$(find adaptive_viewutility)/../output/benchmark.csv"/>
        <param name="num_experiments" value="10"/>
        <param name="max_experiment_duration" value="200"/>
        <param name="result_directory" value="$(find adaptive_viewutility)/../output"/>
    </node>

    <group if="$(arg visualization)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find adaptive_viewutility)/launch/config_planner.rviz" />
    </group>
</launch>
